stl/inc/memory

```diff
@@ -2188,7 +2213,7 @@ _NODISCARD enable_if_t<is_unbounded_array_v<_Ty>, shared_ptr<_Ty>> make_shared(c
     // make a shared_ptr to an unbounded array
     using _Refc    = _Ref_count_unbounded_array<_Ty>;
     const auto _Rx = _Allocate_flexible_array<_Refc>(_Count);
-    _Global_delete_guard _Guard{_Rx};
+    _Global_delete_guard<_Refc> _Guard{_Rx};

```

Comment:
CTAD for aggregates is taunting you again.

Reply:
Not sure if you're looking at the most recent commit - I did change this line. But yes, wish I had CTAD for aggregates here.